screwdriver.yaml

```diff
@@ -3,6 +3,8 @@ shared:
 
 jobs:
     main:
+        annotations:
+            screwdriver.cd/ram: HIGH
```

Comment:
try `TURBO`?

Reply:
I try to add `NODE_OPTIONS`. I think `HIGH` is enough.